### ISSUE TITLE
LSTInputListing labels are not showing up in auto-complete

### ISSUE THREAD

**Issue by patrykgz** [overleaf/overleaf#968]
## Steps to Reproduce

1. Create new \lstinputinsting or /begin{lstlisting} with a [label={lst:mylisting}]
2. Try to reference using \ref{lst:mylisting}

## Expected Behaviour

Overleaf should see the label={lst:mylisting} and add it to the autocomplete dictionary.

## Observed Behaviour

Autocomplete ignores the label definition

## Context

I am trying to reference a listing, but I am unable to do this using autocomplete

## Analysis

I am not sure in this code base how the autocomplete feature works.

## Related
Someone on the stackexchange identified this in 2018, but I can't find the comment as of right now.

**Comment by Amueller36** [comment]
This...

**Comment by patrykgz** [comment]
@aeaton-overleaf was this merged?

**Comment by aeaton-overleaf** [comment]
@patrykgz Yes, a fix for this has been merged, but will take a day or so to arrive in this repository.
